### ISSUE TITLE
Feature Request: Pen mode

### ISSUE THREAD

**Issue by PnkMyo** [rsn8887/uae4all2#41]
Hi ! I was wondering about a "pen mode" for this software, where instead of controlling the mouse relative with the touch screen, you can just use your pen to tap wherever you want. Maybe if you hold down, it can drag ? This would be good for art software !

**Comment by rsn8887** [comment]
Yes unfortunately it is impossible because of the way the Amiga hardware works. There is no concept of absolute mouse position. If you would get it right for one game or app, it will be wrong for another one.
